sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListFileVersionBatchDeleteJobProgress.md

```diff
@@ -0,0 +1,85 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spolistfileversionbatchdeletejobprogress
+applicable: SharePoint Online
+title: Get-SPOListFileVersionBatchDeleteJobProgress
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Get-SPOListFileVersionBatchDeleteJobProgress
+
+## SYNOPSIS
+
+Gets the progress of a file version batch trim job for a site collection.
+
+## SYNTAX
+
+```powershell
+Get-SPOListFileVersionBatchDeleteJobProgress [-Site] <SpoSitePipeBind>
+ [List] <SpoListPipeBind>
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets the progress of a file version batch trim job for a document library.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Get-SPOListFileVersionBatchDeleteJobProgress -Site https://contoso.sharepoint.com/sites/site1 -List "Documents"
+```
+
+Example 1 gets the progress of a file version batch trim job for a document library.
```

Comment:
>for a document library. [](http://example.com/codeflow?start=61&length=23)

`for a document library called "Documents"`

Reply:
Also updated other places using "Documents" as document library in the example